.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Terminus Rsync Plugin
+on: push
+jobs:
+  checkout_lint:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
```

Comment:
Should we use a matrix for different php versions?

Reply:
Not for this job. But I just added it to the functional tests job. Thanks!